data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OffsetManagerTest.java

```diff
@@ -116,47 +125,49 @@ public void shouldNotCommitAndNotGoOutOfBounds() {
 
     @Test
     public void shouldCommitAfterSendingEventsOrderedOnTheSamePartitionWithInducedFailure() {
-        assertThatOffsetCommittedWithFailures(List.of(new TopicPartition("aaa", 0)), (offsetStrategy, failureFlag) -> {
-                    offsetStrategy.recordReceived(record("aaa", 0, 0));
-                    failureFlag.set(true);
-                    for (int i = 0; i < 10; i++) {
-                        var rec = record("aaa", 0, i);
-                        offsetStrategy.successfullySentToSubscriber(rec);
-                    }
-                    failureFlag.set(false);
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 10));
-                })
+        assertThatOffsetCommittedWithFailures(
+                        List.of(new TopicPartition("aaa", 0)), 0, (offsetStrategy, failureFlag) -> {
+                            offsetStrategy.recordReceived(record("aaa", 0, 0));
+                            failureFlag.set(true);
+                            for (int i = 0; i < 10; i++) {
+                                var rec = record("aaa", 0, i);
+                                offsetStrategy.successfullySentToSubscriber(rec);
+                            }
+                            failureFlag.set(false);
+                            offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 10));
+                        })
                 .containsEntry(new TopicPartition("aaa", 0), 11L);
     }
 
     @Test
     public void shouldCommitInAMixedOrderWithInducedFailure() {
-        assertThatOffsetCommittedWithFailures(List.of(new TopicPartition("aaa", 0)), (offsetStrategy, failureFlag) -> {
-                    offsetStrategy.recordReceived(record("aaa", 0, 0));
+        assertThatOffsetCommittedWithFailures(
+                        List.of(new TopicPartition("aaa", 0)), 0, (offsetStrategy, failureFlag) -> {
+                            offsetStrategy.recordReceived(record("aaa", 0, 0));
 
-                    // Order:
-                    // 0 2 1
-                    // flip failure flag
-                    // 4 3
-                    // flip failure flag
-                    // 6 5
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 0));
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 2));
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 1));
-                    failureFlag.set(true);
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 4));
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 3));
-                    failureFlag.set(false);
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 6));
-                    offsetStrategy.successfullySentToSubscriber(record("aaa", 0, 5));
-                })
+                            // Order:
```

Comment:
let's avoid formatting on larger / complex PRs

Reply:
reverted

Reply:
Seems codegen wants it :shrug: 
`/home/prow/go/src/knative.dev/eventing-kafka-broker is out of date. Please run hack/update-codegen.sh` in https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative-extensions_eventing-kafka-broker/4336/build-tests_eventing-kafka-broker_main/1915756775391367168